server/etcdserver/txn/metrics.go

```diff
@@ -39,13 +39,29 @@ var (
 		Buckets: prometheus.ExponentialBuckets(0.0001, 2, 20),
 	},
 		[]string{"version", "op", "success"})
+	rangeSec = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: "etcd",
+		Subsystem: "server",
+		Name:      "range_duration_seconds",
+		Help:      "The latency distributions of txn.Range",
+
+		// lowest bucket start of upper bound 0.0001 sec (0.1 ms) with factor 2
+		// highest bucket start of 0.0001 sec * 2^19 == 52.4288 sec
+		Buckets: prometheus.ExponentialBuckets(0.0001, 2, 20),
+	},
+		[]string{"op", "success"})
```

Comment:
The metrics name already clarifies that it's "range" operation.
```suggestion
		[]string{"success"})
```

Reply:
Fixed. Thanks